parsers/jscript.c

```diff
@@ -427,6 +427,17 @@ static void makeFunctionTag (tokenInfo *const token, vString *const signature, b
 	makeFunctionTagCommon (token, signature, generator, false);
 }
 
+static bool isClassName (tokenInfo *const name)
+{
+	char * p = strrchr(name->string->buffer, '.');
```

Comment:
Please, access the internal of vString directly only when you have a strong reason.
Instead, let's use vStringValue like:
```
vStringValue (name->string);
```

Reply:
Thanks!  That's much better.